sdk/appcenter-crashes/src/main/java/com/microsoft/appcenter/crashes/Crashes.java

```diff
@@ -982,6 +982,7 @@ public void run() {
                         iterator.remove();
                         removeAllStoredErrorLogFiles(id);
                     }
+                    ErrorLogHelper.cleanDirectory(ErrorLogHelper.getPendingMinidumpDirectory());
```

Comment:
I think it's better to use a method without a parameter here since it's not used anywhere else and more clear for a reader. Also, we have a method `FileManager.deleteDir` which can be confused with this one.
e.g. `ErrorLogHelper.cleanPendingMinidumps()`

Reply:
My points to use the directory as a parameter:
- Easier to test
- Reusable (we could (and maybe even should) move this function into FileManager class).

Any other opinions?

Reply:
Create a function `ErrorLogHelper.cleanPendingMinidumps() `and use newly created `FileManager.cleanDir(File dir)` inside. In this case, we have pros from both ideas:
- Easy to test (we test mostly` FileManager.cleanDir`);
- Easy to read (no parameters in `ErrorLogHelper.cleanPendingMinidumps)`;
- Possible reuse of function in FileManager.